### PR TITLE
Add a landing page for non-logged in users

### PR DESCRIPTION
**Pull request by ggeisler**
I'm creating this as a draft because:

- I want to get thoughts on whether this seems like reasonable content for a landing page a non-logged in user would see
- I touched a controller and routes and might not have done that part in the most ideal way (e.g. I named the new stuff `public`, which might not be ideal)
- There are a couple of unfinished loose ends

The goal for this is to provide a bit more of an informational landing page for non-logged in users. It's pretty basic now but we could obviously refine as Mise evolves. With this in place there will be some related things to think about, such as should a non-logged in user be able to see all of the Mise UI they currently see if they select a featured workspace? (I think it's helpful to let the non-logged in user see what a workspace is, etc., but there might be some things we want to hide from  them when they venture past the landing page.)

Screen Shot 2021-05-26 at 9 06 54 AM

---

Things unfinished:

- [x] The `Sign up` button should link to the new user form but I'm not sure what the best way to do that is
- [ ] I think I'd like the gather, arrange, share icons to be more muted (e.g., `text-secondary` color?) but I cannot figure out how to change the color of the icons. The way of doing icons using the `icons.css` file is new to me and no matter where I try changing the color, I get no change.

To finish this up, I'm happy with any approach the developers want to  go with:

- Pair with me and tell me how to fix up the remaining stuff
- Someone can take over this branch and push up a more complete version
- Someone can just use this PR as a design suggestion and start a fresh PR and copy and paste stuff from here as is helpful

### CODE REVIEW

**Comment by anarchivist** [comment]
This is great. I think one thing that would be useful to do is also to provide a contact email address (like a team list) somewhere on the page. I can create a Mailman list to capture traffic, but it might be a good way to have people reach out if they have questions.

I suppose that opens up a broader question about whether at some point we want to incorporate a feedback form, too.

**Comment by ggeisler** [comment]
@anarchivist Yeah, those ideas relate to something I briefly fiddled with but decided to put off in the interest of getting a draft done: creating a custom version of the top navbar for the logged-out state, where we could add some additional information links. So that could be a place to put the contact and/or feedback links you suggest, plus perhaps links to the IIIF and Mirador informational landing pages.

**Comment by anarchivist** [comment]
Thanks! From standup, we decided that a contact link is probably adequate for now. The new email list is `mise-project-team@lists.stanford.edu`.

File: app/views/home/public.html.erb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<div class="px-4 py-5 mb-4 text-center">
+  <h1 class="display-5 fw-bold">Mise</h1>
+  <div class="col-lg-6 mx-auto">
+    <p class="lead mb-4">An exploratory-stage application aimed at curators, students, and scholars, Mise provides a collaborative environment for annotation-oriented projects using the Mirador IIIF viewer.</p>
+    <div class="d-grid gap-2 d-sm-flex justify-content-sm-center">
+      <button type="button" class="btn btn-primary btn-lg px-4 gap-3">Sign up</button>
```
Comment (cbeer) [inline app/views/home/public.html.erb:6]:
```suggestion
      <%= link_to 'Sign up', new_user_registration_path, class: 'btn btn-primary btn-lg px-4 gap-3' %>
```

Maybe?

Comment (ggeisler) [reply app/views/home/public.html.erb:6]: Yep, that works, thanks!

**Comment by ggeisler** [comment]
I did figure out how the change the icon color. I tried the same thing I thought I tried before (updating the `fill` value in the `icons.css` file, but it worked this time. Not sure this is the ideal approach (you can't reuse the same icon and color it differently in different contexts this way) but it does change the color.

**Comment by ggeisler** [comment]
I decided to not worry about the icon color for now. I also suggest we merge this and do a separate PR for updating the navbar with the links discussed above (mostly because I can easily do a mockup for the navbar and let someone else make that update and ensure it's done the best way).

**Pull request merged**